### PR TITLE
Ndc api

### PR DESCRIPTION
**Pull request by beardedfinch**

### CODE REVIEW

File: src/constants/api.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const API_LINK: string = "https://api.fda.gov"
+const API_LINK: string = "https://api.openfda-ndc-demo.semanticbits.com"
```
Comment (dkrylovsb) [inline src/constants/api.jsx]: @beardedfinch may need to be updated

**Comment by dkrylovsb** [comment]
lgtm

File: src/constants/api.jsx (hunk 1 above)
Comment (beardedfinch) [reply src/constants/api.jsx]: Fixed this and a couple other deploy issues.

File: src/pages/apis/drug/ndc/_infographic_definitions.json [hunk 2]
```diff
@@ -0,0 +1,140 @@
+{
+  "globalDefs": {
+    "startYear": 2012,
+    "api": "/drug/enforcement",
```
Comment (dkrylovsb) [inline src/pages/apis/drug/ndc/_infographic_definitions.json]: @beardedfinch Is this correct?

File: src/pages/apis/drug/ndc/understanding-the-api-results/index.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import examples from './_examples.json'
+
+class IndexRoute extends React.Component {
+  render() {
+    const example: string = JSON.stringify(examples["count"], null, '  ') || ''
+
+    return (
+      <section className="doc-content">
+        <h2>Understanding the Query Results</h2>
+        <p>For <code>search</code> queries (such as: <a href="https://api.fda.gov/drug/ndc.json?limit=1">https://api.fda.gov/drug/ndc.json?limit=1</a>), the <code>results</code> section includes matching enforcement reports returned by the API.</p>
```
Comment (dkrylovsb) [inline src/pages/apis/drug/ndc/understanding-the-api-results/index.jsx]: @beardedfinch "enforcement"

**Pull request merged**